### DISCUSSION TITLE
Request for react native support

### DISCUSSION THREAD

**Discussion by psychoSherlock** [dicebear/dicebear#177] [Ideas]
Its hard to render SVG images in react native so it would be really helpfull if png images were provided.

**Comment by FlorianKoerner** [comment]
Hi Athul. Thanks for your suggestion. It may be a duplicate of: https://github.com/dicebear/dicebear/discussions/171

**Comment by FlorianKoerner** [comment]
It is now officially supported.

https://github.com/dicebear/dicebear/discussions/179
